test/shape/data/in-house/Makefile.sources

```diff
@@ -71,5 +71,11 @@ TESTS = \
 	tests/zero-width-marks.tests \
 	$(NULL)
 
+PLAT_SHAPER_TESTS = \
+	tests/coretext.tests \
+	tests/uniscribe.tests \
+	tests/directwrite.tests \
+	$(NULL)
+
```

Comment:
This should be (I think):
```
if HAVE_CORETEXT
 	TESTS += tests/coretext.tests
endif
```

etc.

Reply:
Here we go... let's hope this does not trip up CMake :).

Reply:
We don’t enable tests with CMake build IIRC.